editors/vscode/language-configuration.json

```diff
@@ -0,0 +1,104 @@
+{
+	"comments": {
+		"lineComment": "//",
```

Comment:
Nit: at least at present, comments are `#`, contrary to the tutorial.

Reply:
Yes. I just copy/pasted this file. The grammar uses `#`: https://github.com/dada-lang/dada/pull/99/files#diff-5131d9d0df4506ca526c37bc2d267def6694ca7ffbde11a41adf4e207c7ce37cR10-R12